documentation-site/components/overrides.js

```diff
@@ -84,6 +84,14 @@ class Overrides extends React.Component {
             },
           })}
         </Block>
+        <Block as="p" font="font400" marginTop="scale900">
```

Comment:
should we use the warning inline notifications instead?

Reply:
You mean the `<Blockquote>` or the baseui notification component? I don't think it should be that super visible since it's not even 100% related to this specific location - it's more bold and detailed in the Overrides page.

Reply:
yeah, I meant the base notification component - but yeah, your point makes perfect sense, as always, I was just being the devil advocate 😈